spec/controllers/password_reset_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+describe PasswordResetController do
+  describe 'POST confirm_password_reset' do
+    context 'with blank email' do
+      it 'redirects to forgot password page' do
+        post :confirm_password_reset
+        expect(response).to redirect_to forgot_password_path
+      end
+
+      it 'shows an error message' do
+        post :confirm_password_reset
+        expect(flash[:error]).to eq('Please enter a user email.')
+      end
+    end
+
+    context 'with existing user email' do
+      after { ActionMailer::Base.deliveries.clear }
+      
+      it 'sends an email to user if regiestered' do
```

Comment:
You don't need this "if" in the description, or the next one. Your context says that these specs are for existing users.